app/main.py

```diff
@@ -2,0 +1,25 @@
+from app.cafe import Cafe
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+def go_to_cafe(friends: list, cafe: Cafe) -> str:
+    masks_to_buy = 0
+
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
```

Comment:
The 'visit_cafe' method might not be clear for people who see your code for the first time. It would be better to rename it to something more descriptive, like 'check_entry_requirements'.